neard/Cargo.toml

```diff
@@ -48,3 +48,7 @@ testlib = { path = "../test-utils/testlib" }
 adversarial =  ["near-client/adversarial", "near-network/adversarial"]
 expensive_tests = ["near-client/expensive_tests", "near-epoch-manager/expensive_tests", "near-chain/expensive_tests"]
 metric_recorder = ["near-network/metric_recorder", "near-client/metric_recorder"]
+
+[[bin]]
+path = "src/near.rs"
+name = "near"
```

Comment:
shouldn't this be `neard` as well?

Reply:
@bowenwang1996 I believe this is for the backward compatibility. We don't want to break all the tooling at once, thus we will have `near` and `neard` binaries in nearcore while we migrate our tooling.

Reply:
@ailisp I wonder if we can use `path = "src/main.rs"` for this binary as well, this way we don't need to play around `lib.rs` + `main.rs` & `near.rs`.

Reply:
@frol Doing that cargo will give a warning, and currently we have "fail on warning" on ci

Reply:
Warning is 
`warning: /home/bo/workspace/nearcore2/neard/Cargo.toml: file found to be present in multiple build targets: /home/bo/workspace/nearcore2/neard/src/main.rs`
If doing:
```
[[bin]]
path = "src/main.rs"
name = "neard"

[[bin]]
path = "src/main.rs"
name = "near"
```
With only use 
```
[[bin]]
path = "src/main.rs"
name = "near"
```
cargo only compile it to near but no neard

Reply:
We can workaround the warning my using:

```toml
[[bin]]
path = "src/main.rs"
name = "neard"

[[bin]]
path = "src/../src/main.rs"
name = "near"
```

This is a hack, but it is not as intrisuve as the need to have `lib.rs` + two `main.rs` and `near.rs`.

WDYT?

Reply:
:joy: this is smart, I definitely won't report "cargo cannot detect `path = "src/../src/main.rs"` replicate with `path = "src/main.rs"`" to cargo issues